rasa/cli/interactive.py

```diff
@@ -68,57 +83,30 @@ def _set_not_required_args(args: argparse.Namespace) -> None:
     args.store_uncompressed = False
 
 
-def interactive_core(args: argparse.Namespace):
-    _set_not_required_args(args)
-
-    if args.model is None:
-        zipped_model = train.train_core(args)
-    else:
-        zipped_model = get_provided_model(args.model)
-
-    perform_interactive_learning(args, zipped_model)
-
-
-def perform_interactive_learning(args, zipped_model):
+def perform_interactive_learning(args: argparse.Namespace, zipped_model: Text) -> None:
     from rasa.core.train import do_interactive_learning
 
-    if zipped_model and os.path.exists(zipped_model):
-        args.model = zipped_model
+    args.model = zipped_model
 
-        with model.unpack_model(zipped_model) as model_path:
-            args.core, args.nlu = model.get_model_subdirectories(model_path)
-            stories_directory = data.get_core_directory(args.data)
-
-            args.endpoints = get_validated_path(
-                args.endpoints, "endpoints", DEFAULT_ENDPOINTS_PATH, True
+    with model.unpack_model(zipped_model) as model_path:
+        args.core, args.nlu = model.get_model_subdirectories(model_path)
+        if args.nlu is not None and args.core is None:
```

Comment:
Isn't this check enough?
```suggestion
        if args.core is None:
```